main/release/utils/github.py

```diff
@@ -16,41 +16,57 @@ class GitHub:
 
     def __init__(self):
         self.token = os.environ.get('GITHUB_TOKEN')
-        if os.environ.get('GITHUB_EVENT_NAME') != 'release':
-            raise GitHubException('The action was not triggered on release event')
+
+        ALLOWED_GITHUB_ACTIONS = set(['release', 'workflow_dispatch'])
+        github_event = os.environ.get('GITHUB_EVENT_NAME')
+        if github_event not in ALLOWED_GITHUB_ACTIONS:
+            raise GitHubException(f"The action was not triggered on {ALLOWED_GITHUB_ACTIONS} events (is: '{github_event}')")
+
         with open(os.environ.get('GITHUB_EVENT_PATH')) as file:
             self.event = json.load(file)
 
     def get_release_request(self) -> ReleaseRequest:
         repo = self._get_repository()["full_name"]
         organisation, project = repo.split("/")
-        version = self._get_release()['tag_name']
+
+        # Attempt to get the version number. Depending on the event type,
+        # it can be the release tag or an user provided input.
+        release = self._get_release()
+        version = self.event['inputs']['version'] if release is None else release['tag_name']
```

Comment:
Looks like the version is missing as an input of the releasability check workflow

Reply:
> I don't see how the user can run the releasability check without providing the version as input.

@drautureau-sonarsource You are correct. It is required.